paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/MultiTableCommittable.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.flink.sink;
+
+import org.apache.paimon.catalog.Identifier;
+
+/**
+ * MultiTableCommittable produced by {@link PrepareCommitOperator}. This type of Committable will
+ * only be produced by multiplexed operators that handles data from multiple tables. Thus, the
+ * database, table, and commit user of each table is included in the committable.
+ */
+public class MultiTableCommittable extends Committable {
+
+    private final String database;
+    private final String table;
+    private final String commitUser;
+
+    public MultiTableCommittable(
+            String database,
+            String table,
+            String commitUser,
```

Comment:
Do we need this one? We can set same UUID user `StreamWriteBuilder.withCommitUser` to writer and committer.

Reply:
Yes, it's possible. There are several places where commit user is used.

1. When the write operator is assembly into job graph, a random uid is set to be the commit user id.

2. When the job is deployed and running, it will try to recover the previous commit user from Flink state to make the commit user consistent accross different Flink job attempts.

3. When the CommitterOperator do commit, it will use create the committer with commit user.


So in this case we user the same commit user for all tables.